dpnp/tests/test_sycl_queue.py

```diff
@@ -580,6 +580,51 @@ def test_1in_1out(func, data, device):
     assert_sycl_queue_equal(result_queue, expected_queue)
 
 
+@pytest.mark.parametrize(
+    "op", ["bitwise_count", "bitwise_not"], ids=["bitwise_count", "bitwise_not"]
```

Comment:
I believe there is no need to list IDs, because the value is string:
```suggestion
    "op", ["bitwise_count", "bitwise_not"]
```